OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -120,6 +120,7 @@ protected void onResponse(@NonNull DiscussionThread responseBody) {
     }
 
     private void loadThreadResponses() {
+        setScreenTitle();
```

Comment:
this function shouldn't be called here. Setting the screen's title has nothing to do with loading thread responses.
Call `setScreenTitle` explicitly wherever you need to set the title only.